mso/resource_mso_schema_template_anp_epg_subnet.go

```diff
@@ -228,7 +260,22 @@ func resourceMSOSchemaTemplateAnpEpgSubnetCreate(d *schema.ResourceData, m inter
 		shared = tempVar.(bool)
 	}
 
-	schemaTemplateAnpEpgSubnetApp := models.NewSchemaTemplateAnpEpgSubnet("add", "/templates/"+templateName+"/anps/"+anpName+"/epgs/"+epgName+"/subnets/-", ip, scope, shared)
+	querier := false
```

Comment:
Why is this explicitly assigned false and not defined as bool?

Reply:
wanted to keep consistent with shared attribute. should not make a difference in behaviour because define a var as bool is also equal to false

Reply:
The main concern is if these attributes are set to True to begin with then they might be forced to False during a second apply (update) if some other attribute is updated.
As discussed d.GetOk won't be influenced by the explicit assignment and will take the value from the config. directly.